Rules/UseConsistentIndentation.cs

```diff
@@ -220,14 +220,13 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                 // Check if the current token matches the end of a PipelineAst
                 var matchingPipeLineAstEnd = pipelineAsts.FirstOrDefault(pipelineAst =>
                         PositionIsEqual(pipelineAst.Extent.EndScriptPosition, token.Extent.EndScriptPosition)) as PipelineAst;
-
                 if (matchingPipeLineAstEnd == null)
                 {
                     continue;
                 }
-
-                bool pipelineSpansOnlyOneLine = matchingPipeLineAstEnd.Extent.StartLineNumber == matchingPipeLineAstEnd.Extent.EndLineNumber;
-                if (pipelineSpansOnlyOneLine)
+                bool pipelinesSpansOnlyOneLine = matchingPipeLineAstEnd.PipelineElements[0].Extent.StartLineNumber ==
```

Comment:
```suggestion
                bool pipelinesSpanOnlyOneLine = matchingPipeLineAstEnd.PipelineElements[0].Extent.StartLineNumber ==
```

Reply:
Assuming this name change is intentional. The AST type is `PipelineAst` and conceptually in the syntax, a "pipeline" is a instance of `cmd1 | cmd2 | cmd3`, so that pipeline*s* would be something like `cmd1 | cmd2; cmd3 | cmd4`, where there are multiple statements. So I'm thinking `pipelineSpansOnlyOneLine` accords best to the AST naming scheme

Reply:
Addressed (could not use suggestion as I had to adapt name where variable is used as well